manifests/config.pp

```diff
@@ -30,7 +30,7 @@
     $::redis::log_dir:
       ensure => directory,
       group  => $::redis::service_group,
-      mode   => $::redis::config_dir_mode,
```

Comment:
Ja ik ben de log_dir_mode in init.pp vergeten.
Kga die toevoegen, pushen en zal nieuwe pull request doen

On 05/05/2015 11:52 AM, Tom De Vylder wrote:
>
> In manifests/config.pp <https://github.com/arioch/puppet-redis/pull/31#discussion_r29655549>:
>
> > @@ -30,7 +30,7 @@
> >      $::redis::log_dir:
> >        ensure => directory,
> >        group  => $::redis::service_group,
> > -      mode   => $::redis::config_dir_mode,
>
> Just add $log_dir_mode instead of renaming $config_dir_mode.
>
> —
> Reply to this email directly or view it on GitHub <https://github.com/arioch/puppet-redis/pull/31/files#r29655549>.
>